### ISSUE TITLE
Is it possible to train and predict without letterbox image preprocessing

### ISSUE THREAD

**Issue by thomassajot** [ultralytics/yolov5#10269]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have designed my input images to be of shape 704x1600 (both a multiple of 64 ,11 and 25 resp).  However when trainning on this dataset with `--imgs 1600 --rect`, the model is fed modified images of size  768x1664. 
How come ? Is it possible not to have the images being modified?

**Comment by glenn-jocher** [comment]
@thomassajot val adds padding, you can update the vallaoder to pad=0.0 if you want.

**Comment by thomassajot** [comment]
You are right, I printed the values in train and the images are not padded to a larger size. But there are in the validation set. 
Why is the validation set padded?

https://github.com/ultralytics/yolov5/blob/40bb8030f8468eb7145ff648588aa5f96e32447c/train.py#L218

**Comment by Tongfengyu** [comment]
I guess it's a method to overcome overfitting

**Comment by thomassajot** [comment]
I don't think changing the validation image format contributes to preventing or measuring overfitting.

**Comment by glenn-jocher** [comment]
@thomassajot you are correct! The padding of validation images doesn't directly contribute to preventing or measuring overfitting. It's meant to ensure compatibility with the model's input size requirements during validation. If you'd like to remove the padding for validation images, you can modify the [vallist padding code](https://github.com/ultralytics/yolov5/blob/40bb8030f8468eb7145ff648588aa5f96e32447c/train.py#L213) to 0.0.
